samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Shared/App.xaml.cs

```diff
@@ -117,7 +119,7 @@ protected override async void OnLaunched(XamlLaunchActivatedEventArgs e)
 				_window.Activate();
 
 				// Uncomment this line to see the splashscreen for longer
-				// await Task.Delay(10000);
+				await Task.Delay(10000);

```

Comment:
Should we uncomment this by default ?

Reply:
oh no no... i was just testing
thanks god you caught this